tests/persist.test.js

```diff
@@ -774,6 +774,48 @@ test('transformers order', async () => {
   });
 });
 
+test('migrations', async () => {
+  // ARRANGE
+  const memoryStorage = createMemoryStorage({
+    '[EasyPeasyStore][0]': {
+      foo: "foo-updated",
+      migrationConfict: "error"
+    },
+  });
+
+  const makeStore = () =>
+    createStore(
+      persist(
+        {
+          foo: {
+            bar: "bar",
+          },
+          bar: "bar",
+        },
+        {
+          storage: memoryStorage,
+          migrations: {
+            migrationVersion: 1,
```

Comment:
Didn't want to use `version` here so as to not confuse the top-level `persistKey` version. `migrationVersion` seems sufficiently unambiguous